ui/desktop/app/routes/scopes/scope/projects/targets/index.js

```diff
@@ -84,29 +84,26 @@ export default class ScopesScopeProjectsTargetsIndexRoute extends Route {
     { search, scopes, availableSessions, types, page, pageSize },
     transition,
   ) {
-    const { id: scope_id } = this.modelFor('scopes.scope');
-    await this.getAllTargets(transition, scope_id);
+    const orgScope = this.modelFor('scopes.scope');
+    // orgFilter used to narrow down resources to only those under
+    // the current org scope if org is not global
+    const orgFilter = `("/item/scope/parent_scope_id" == "${orgScope.id}")`;
```

Comment:
I'm not familiar with the filter syntax but are the parentheses needed?

Reply:
Actually it seems me don't. I was basing this off of the filter we use for filtering by project type but I have removed it and it doesn't seem to be an issue.